relay/sources/apache/rudder-apache-relay-ssl.conf

```diff
@@ -27,7 +27,7 @@ SSLSessionTickets       off
 # ----------------------
 
 # List of allowed certificates
-SSLCACertificateFile /var/rudder/lib/ssl/nodescerts.pem
+SSLCACertificateFile /var/rudder/lib/ssl/allnodescerts.pem
```

Comment:
this file only exists on the root server IIRC, this won't work

Reply:
it contains all the nodes of the infrastructure, we want to use nodescerts on each relay which only contains the managed nodes by relay

Reply:
in this case we need to add linux agents to nodescerts.pem